jupyter-extension/jupyterlab_pachyderm/tests/test_integrations.py

```diff
@@ -401,24 +400,21 @@ def test_pps(dev_server, simple_pachyderm_env):
     client, repo_name, pipeline_name = simple_pachyderm_env
     path = TEST_NOTEBOOK.relative_to(Path.cwd())
     image = "combinatorml/jupyterlab-tensorflow-opencv:0.9"
-    input_spec = dict(pfs=dict(repo=repo_name, glob="/*"))
     data = dict(
-        apiVersion='sameproject.ml/v1alpha1',
-        environments=dict(default=dict(image_tag=image)),
-        metadata=dict(name=pipeline_name, version='0.0.0'),
-        notebook=dict(requirements=''),
-        run=dict(name=pipeline_name, input=json.dumps(input_spec)),
+        pipeline_name=pipeline_name,
+        image=image,
+        input_spec=dict(pfs=dict(repo=repo_name, glob="/*"))
     )
     r = requests.put(f"{BASE_URL}/pps/_create/{path}", data=json.dumps(data))
     assert r.status_code == 200
     assert next(client.inspect_pipeline(pipeline_name))
 
 
-@pytest.mark.parametrize('excluded_field', SAME_METADATA_FIELDS)
+@pytest.mark.parametrize('excluded_field', ('pipeline_name', 'image', 'input_spec'))
```

Comment:
Future work: These fields should still be captured in a variable to reduce duplication